test/functional/apps/dashboard/group5/embed_mode.ts

```diff
@@ -71,12 +71,7 @@ export default function ({
         await testSubjects.existOrFail('globalQueryBar');
       });
 
-      /**
-       * Skipping all render tests for now - there is a problem where the locally generated screenshots do not align with the
-       * CI screenshots due to (possibly) pixel density or something similar. This fix is super important to get in so we will
-       * have to resolve the issue with these new tests *after* FF for 8.9/8.8.2
-       */
-      it.skip('renders as expected', async () => {
+      it('renders as expected', async () => {
```

Comment:
Nit: Maybe we should add a comment here saying that there is a chance that these tests will fail when run locally?

Reply:
Ah, I totally meant to do that and then managed to forget somehow 😆 Will do